### ISSUE TITLE
What is the actual impact of ”scope" property?

### ISSUE THREAD

**Issue by yutaro113** [MicrosoftDocs/msteams-docs#5386]
Hello, 
There is "scope" property for each functionality. What is the impact of this selection to application behavior? (In other words, how user  can check this scope is working properly?)
It would be great help to have this point documented more concretely.
https://docs.microsoft.com/ja-jp/microsoftteams/platform/resources/schema/manifest-schema#bots

I have tested with Bot feature.
1, I did not select any scope for example and publish this application.
![image](https://user-images.githubusercontent.com/36602021/156963526-e55bdabd-11fb-421a-9179-89eb348d852d.png)

2, But I can still add this app to Group Chat / Personal Chat / Team.
![image](https://user-images.githubusercontent.com/36602021/156963634-245c486b-8d27-4769-95b2-e1c92b2512b1.png)

I could add this app to 1 - 1 chat and mention it.
![image](https://user-images.githubusercontent.com/36602021/156963810-1b0eb410-7917-453d-86a0-156f350f01cc.png)

from doc explanation, I expect this property control the place we can use it but is it expected behavior?
Or this property is for another behavior control?

Thanks in advance!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0f94a488-d636-92b3-21eb-c88744c8f272
* Version Independent ID: 7b3413f4-9ade-accf-5cb2-83fead11a32a
* Content: [Manifest schema reference - Teams](https://docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema#bots)
* Content Source: [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/main/msteams-platform/resources/schema/manifest-schema.md)
* Product: **msteams**
* GitHub Login: @surbhigupta
* Microsoft Alias: **lajanuar**

**Comment by Meghana-MSFT** [comment]
Scope defines the place where you can use that particular feature. In the manifest schema its mentioned that Scope is a required property. 
![image](https://user-images.githubusercontent.com/81138249/157030594-2573c107-d353-4371-9552-0c728883b976.png)


We tried the above scenario and we do not get add option until we include the scopes in manifest. Attaching screenshots below.

Without Scope : 
![image](https://user-images.githubusercontent.com/81138249/157031143-817fc4ed-f10a-4543-8307-0d58717a1ecc.png)

Personal Scope:
![image](https://user-images.githubusercontent.com/81138249/157030461-b853ef01-fc2b-4c66-b900-c303a0c57255.png)

Can you please try once again? Meanwhile we are checking this behavior with the engineering team.

**Comment by yutaro113** [comment]
Hi, @Meghana-MSFT , thank you very much for the quick confirmation!
I could understood expected behavior, but in my environment, even if I clear the cache, app can still be added without scope.

0, Uninstall Custom App 
At this point, no scope is specified for this app.
![image](https://user-images.githubusercontent.com/36602021/157039299-a2911e54-888a-46b6-95fd-1575c991aece.png)

1, Shutdown Teams and Clear cache (delete directory)
![image](https://user-images.githubusercontent.com/36602021/157038837-0486aebc-1149-4ff1-96bb-6267b8a2f33a.png)

2, Restart Teams
3, Try to install the same custom app -> "Add" button to Team/Chat/Meeting is still available.
![image](https://user-images.githubusercontent.com/36602021/157039160-bead6522-0a29-4c98-98ea-97704301325e.png)

Thank you very much!

**Comment by Meghana-MSFT** [comment]
We will check this internally and get back to you.

**Comment by Meghana-MSFT** [comment]
Could you please share your manifest?

**Comment by yutaro113** [comment]
Hi @Meghana-MSFT , thank you very much for the confirmation.
And sorry, when I test it again today, "scope" seems working properly as you mentioned.
May be I have tested with old version app (with full scope)  on organization store whereas I saw newer version app (with no scope) on Developer Portal.... Sorry for the confusion. If issue occurs again, let me reopen this issue.

As of now, I could understand the expected behavior so we can close this issue.
If there's a room for discussion, I think it would be kind for user to describe this expected behavior (how "scope" will impact to actual behavior).

Thank you very much!
